frontend/src/stores.ts

```diff
@@ -67,17 +67,24 @@ export const models: Writable<string[]> = writable([]);
 export const model: Writable<string> = writable(undefined);
 export const metric: Writable<string> = writable(undefined);
 
-export const comparisonModels: Writable<string[]> = writable([]);
+export const comparisonModel: Writable<string> = writable(undefined);
 
 // [column, ascending]
 export const sort: Writable<[ZenoColumn, boolean]> = writable([
 	undefined,
 	true,
 ]);
+export const compareSort: Writable<[ZenoColumn | string, boolean]> = writable([
+	undefined,
+	true,
+]);
 
 export const requestingHistogramCounts: Writable<boolean> = writable(false);
 
 export const slices: Writable<Map<string, Slice>> = writable(new Map());
+export const modelDependSlices: Writable<Map<string, Slice>> = writable(
```

Comment:
rename? or add comment, don't know what this is

Reply:
These slices are created only in comparison tab. 
So when switching to /comparison tab, I check if the slices are "model-depended", which includes predicates that are postdistill or output types.

For example, "has special characters" is a model-depended slice,
So I create two new slices that have different names, "has special characters-model A", "has special characters-model B" from this slice and stored them in this "modelDependSlices" Writable. 
The logic is to separate model-depended slices into two new slices that are easier to perform slice cell selection, and maintain the same logic that every chip in the selectionbar is a single slice.

![image](https://github.com/zeno-ml/zeno/assets/89871868/108fe139-9abf-4072-af1c-ccc90cd97de4)

Reply:
I *think* this makes sense. Can we rename is "slicesDependentOnModels" and add a comment above it explaining briefly what they are for/when they are used?

Reply:
Or better yet, `slicesForComparison`?

Can we also change the chip style to be `has special characters (Model A)`

Reply:
Sure can :)